src/renderer.js

```diff
@@ -148,10 +148,21 @@ const Papa = require('papaparse');
             const decodedPassword = decodeLoginData(login.encryptedPassword);
             const username = decrypt(decodedUsername.data, decodedUsername.iv, key);
             const password = decrypt(decodedPassword.data, decodedPassword.iv, key);
+
+            // add timestamp infos
+            const timeCreated = Math.floor(login.timeCreated / 1000);
```

Comment:
Are these just epoch timestamps? Why not just include the raw millisecond value?

Reply:
Some import methods don't work with millis. For example Keepass doesn't properly handle timestamp with millis : it set the date to year 50000 instead of 2018 ( no I'm not back from the future :smile:  ). What do you think ?

Reply:
I'd still leave it as raw `ms`. You can easily `/ 1000` it using Excel or something if that is the desired format.